docker-compose-mastodon-update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+git stash
+git pull
```

Comment:
That is very interesting and in my tests seemed to work. Thank you for that info, I can really use that in a lot of my scripts. I was wondering if the docs had changed because when I wrote this I'm pretty sure it just said git pull. I'm gonna put this in. I'll leave my personal version as is because I like running latest though. Seems I'm a bold person ;)